backend/cmd/migrate/migrate.go

```diff
@@ -137,9 +139,32 @@ func (m *Migrator) Up() {
 		logger: m.log.Sugar(),
 	}
 
+	return sqlMigrate
+}
+
+func (m *Migrator) Up() {
+	msg := "migration has the potential to cause irrevocable data loss, verify host information above"
+	m.confirmWithUser(msg)
+
+	sqlMigrate := m.setupSqlMigrator()
+
 	// Apply migrations!
-	m.log.Info("applying migrations", zap.String("migrationDir", migrationDir))
-	err = sqlMigrate.Up()
+	m.log.Info("applying up migrations")
+	err := sqlMigrate.Up()
+	if err != nil && err != migrate.ErrNoChange {
+		m.log.Fatal("failed running migrations", zap.Error(err))
+	}
+}
+
+func (m *Migrator) Down() {
+	msg := "Migrating DOWN by ONE version this migration has the potential to cause irrevocable data loss, verify host information above"
+	m.confirmWithUser(msg)
+
+	sqlMigrate := m.setupSqlMigrator()
+
+	// Migrate back by 1
+	m.log.Info("applying migrations down")
+	err := sqlMigrate.Steps(-1)
```

Comment:
let's have the confirmation include which step they're migrating from

Reply:
```
2020-12-02T10:58:28.455-0800	WARN	migrate/migrate.go:95	Migrating DOWN by ONE version from (6 -> 5) this migration has the potential to cause irrevocable data loss, verify host information above

*** Continue with migration? [y/N] y
```